### ISSUE TITLE
How to set state for Bottomsheet fullscreen

### ISSUE THREAD

**Issue by minhson95th** [nativescript-community/ui-persistent-bottomsheet#15]
I try to set state of Bottomsheet to fullscreen by set steps = height of screen of device
``` 
        var heightScreen = Screen.mainScreen.heightDIPs
        setTimeout(() => {
            let actionBarHeight = evt.object.getActualSize().height
            var height = heightScreen - actionBarHeight
            console.log('heightScreen: ' + heightScreen); //640
            console.log('actionBarHeight: ' + actionBarHeight); //56
            console.log('height: ' + height); //584

            if (isIOS) {
                let bottomSafeArea = application.ios.window.safeAreaInsets.bottom
                height= height - bottomSafeArea - 25
            }
            this.appService.bottomSheetSteps = [75, height / 2, height]
        }, 200);
```
But bottomSheet show unexpected. I have try in Samsung Galaxy S6 and Pixel 2 Emulator
<img src="https://github.com/nativescript-community/ui-persistent-bottomsheet/assets/46039866/5bfaba9f-8c75-44e3-ae83-d43539e749ff" width=35% height=40%> <img src="https://github.com/nativescript-community/ui-persistent-bottomsheet/assets/46039866/55f94527-1146-4053-9e88-84179cccfd76" width=35% height=40%>

Expected:

<img src="https://github.com/nativescript-community/ui-persistent-bottomsheet/assets/46039866/cc692d04-b0e1-4b88-a646-6b1ad4f52542" width=35% height=40%>

**Comment by farfromrefug** [comment]
@minhson95th you cant doing that way. The reason is that the persistent bottomsheet is added to the child view beneath the actionbar. So it will never go on top of it. 
For that you need a custom actionbar and add the bottomsheet to the parent of the actionbar

**Comment by minhson95th** [comment]
No, i don't want bottomsheet on top of action bar, i mean how to calculate height of bottomshet from below of actionbar to down for all device, look like picture
<img src="https://github.com/nativescript-community/ui-persistent-bottomsheet/assets/46039866/31da3f1c-c1e9-4e90-a19f-62a03286084a" width="50%" height="50%">

**Comment by farfromrefug** [comment]
@minhson95th you either use the `getMeasuredHeight` of the parent use `Device.mainScreen.heightDips` and removes status bar, action bar, navigation bar...
BTW you should ask that kind of question on Discord, more people could be able to help as it is not a question related to the plugin itself but more to N
